src/cmap/conn.rs

```diff
@@ -120,6 +120,12 @@ pub(crate) struct Connection {
     /// monitoring connections as we do not emit events for those.
     #[derivative(Debug = "ignore")]
     event_emitter: Option<CmapEventEmitter>,
+
+    /// The token callback for OIDC authentication.
+    /// TODO RUST-1497: make this `pub`
```

Comment:
Why make this `pub`? (the struct this is on is only `pub(crate)`)

Reply:
I think you are right. I'll remove the comment.